eduNEXT/drydock
# feat: add poddisruptionbudget

# Description:
This PR add PodDisruptionBudget for `caddy|lms|lms-worker|cms|cms-worker|forum|mfe` deployments

## How to test it?
You can install the last version of drydock and include variable `DRYDOCK_ENABLE_PDB = true` in `config.yml` file, run `tutor config save` or `make config` and syncronize.
For verify PDBs, you can run in your namespace `kubectl get pdb` and you can see:
``` bash
NAME             MIN AVAILABLE   MAX UNAVAILABLE   ALLOWED DISRUPTIONS   AGE
caddy-pdb        30%             N/A               2                     11m
cms-pdb          30%             N/A               2                     11m
cms-worker-pdb   30%             N/A               2                     11m
forum-pdb        30%             N/A               2                     11m
lms-pdb          30%             N/A               2                     11m
lms-worker-pdb   30%             N/A               2                     11m
mfe-pdb          30%             N/A               2                     11m
```
You can verify its operation in the Yamato cluster in the [quince namespace](https://argocd.services.stage.edunext-shipyard.net/applications/argocd/quince.yamato?view=tree&conditions=false&resource=)
